MvvmCross.Android.Support/Design/MvxBottomNavigationView.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+using Android.Content;
+using Android.Runtime;
+using Android.Widget;
+using Android.Util;
+using Android.Views;
+using Android.Support.Design.Widget;
+
+namespace MvvmCross.Droid.Support.Design
+{
+    [Register("mvvmcross.droid.support.design.MvxBottomNavigationView")]
+    public class MvxBottomNavigationView : BottomNavigationView, BottomNavigationView.IOnNavigationItemSelectedListener
+    {
+        public List<MvxMenuItem> ItemsSource = new List<MvxMenuItem>();
```

Comment:
Use ItemsSource like this: https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Platforms/Android/Binding/Views/MvxListView.cs#L73